debug/customization/vector-tile-layer/smk-init.js

```diff
@@ -0,0 +1,298 @@
+                        color: '#c545d3',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    aeroway: {
+                        weight: 1,
+                        fillColor: '#51aeb5',
+                        color: '#51aeb5',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    road: {	
+                        weight: 1,
+                        fillColor: '#f2b648',
+                        color: '#f2b648',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    transit: {
+                        weight: 0.5,
+                        fillColor: '#f2b648',
+                        color: '#f2b648',
+                        fillOpacity: 0.2,
+                        opacity: 0.4,
+                    },
+                    building: {
+                        fill: true,
+                        weight: 1,
+                        fillColor: '#2b2b2b',
+                        color: '#2b2b2b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    water_name: {
+                        weight: 1,
+                        fillColor: '#022c5b',
+                        color: '#022c5b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    transportation_name: {
+                        weight: 1,
+                        fillColor: '#bc6b38',
+                        color: '#bc6b38',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    place: {
+                        weight: 1,
+                        fillColor: '#f20e93',
+                        color: '#f20e93',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    housenumber: {
+                        weight: 1,
+                        fillColor: '#ef4c8b',
+                        color: '#ef4c8b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    poi: {
+                        weight: 1,
+                        fillColor: '#3bb50a',
+                        color: '#3bb50a',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    earth: {
+                        fill: true,
+                        weight: 1,
+                        fillColor: '#c0c0c0',
+                        color: '#c0c0c0',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    }
+                };
+
+                // Monkey-patch some properties for nextzen layer names, because
```

Comment:
Is this necessary? It looks like the copied code was last updated in 2019.

Reply:
Looking at how the vectorTileStyling object is used, you can simply use the correct property names when the object is first declared. For example, 
```
var vectorTileStyling = {
                    buildings: { 
                        fill: true,
                        weight: 1,
                        fillColor: '#06cccc',
                        color: '#06cccc',
                        fillOpacity: 0.2,
                        opacity: 0.4,
                    }, ...
```
Notice that I've replaced building with buildings, there will no longer be a buildings property. You will then have to update the esriStyle object, 
```
esriStyle["City small scale"             ] = vectorTileStyling.buildings;
```

Reply:
I'm not seeing any issues when I make the suggested change, boundaries continue to appear as magenta provided you update the associated property in the esriStyle object. 

The reason I commented on this is that I feel that leaving this 'monkey patch' in place adds unnecessary complexity and is confusing. I feel it's not an example of good coding practice and don't really want to demonstrate this in our code base. I'll leave the final decision up to you.

Reply:
You're right, I'd probably forgotten to update the Esri style as "boundaries" - I can reproduce what I saw earlier by not updating the Esri "boundary" as "boundaries". 

I'll reapply my earlier changes and address the omission.